include/megaapi.h

```diff
@@ -17642,7 +17767,19 @@ class MegaApi
         MegaApiLock* getMegaApiLock(bool lockNow);
 
         /**
-         * @brief Request a list of all Smart Banners available for current user.
+         * @brief Requests a list of all Smart Banners available for current user.
+         *
+         * The response value is stored as a MegaBannerList.
+         *
+         * The associated request type with this request is MegaRequest::TYPE_GET_BANNERS
+         * Valid data in the MegaRequest object received in onRequestFinish when the error code
+         * is MegaError::API_OK:
+         * - MegaRequest::getMegaStringListMap where the keys are two-letter country codes and the
+         *   values a list of calling codes.
```

Comment:
Outdated copy&paste:
```suggestion
         * - MegaRequest::getMegaBannerList: the list of banners
```